templates/js/copilot-plugin-from-scratch/infra/azure.bicep

```diff
@@ -78,3 +78,4 @@ var apiEndpoint = 'https://${functionApp.properties.defaultHostName}'
 // The output will be persisted in .env.{envName}. Visit https://aka.ms/teamsfx-actions/arm-deploy for more details.
 output API_FUNCTION_ENDPOINT string = apiEndpoint
 output API_FUNCTION_RESOURCE_ID string = functionApp.id
+output OPENAPI_SERVER_URL string = '${apiEndpoint}/api'
```

Comment:
So the server url should be xxx/api? If so, I think we should place /api in the repair.yml, i.e. {{OPENAPI_SERVER_URL}}/api, because debug-start-local-tunnel task only outputs the tunnel endpoint and does not know it should append /api to the endpoint.

Reply:
Yes, it should be `xxx/api`. I've placed the `/api` in the repari.yml file and also update the output url in bicep file.